src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    public static void goodbye()
+    {
+        System.out.println("\nBye.Hope to see you again soon!");
+    }
+
+    public static void echo(String content)
+    {
+        System.out.println(content+"\n");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);*/
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
```

Comment:
Perhaps this code block could be extracted into a new method? It will help improve code readability by following SLAP. It could be similar to your goodbye() method.